tests/fixtures/new/.zk/config.toml

```diff
@@ -10,6 +10,10 @@ filename = "{{date now '%d-%m'}}"
 template = "empty.md"
 filename = "{{now}}"
 
+[group.id.note]
+template = "empty.md"
```

Comment:
Not partial on how to test the id showing up in the templates. I did a new group to not affect the other tests but maybe just injecting the id into the default template would be better?

Reply:
No I think you did well adding a dedicated group.